torchbench.py

```diff
@@ -238,9 +238,9 @@ def speedup_experiment(args, model_iter_fn, model, example_inputs):
     timings = np.zeros((args.repeat, 2), np.float64)
     for rep in range(args.repeat):
         # interleave the runs to handle frequency scaling and load changes
-        timings[rep, 0] = timed(model, model_iter_fn, example_inputs)
+        timings[rep, 0] = timed(model, model_iter_fn, example_inputs, times=30)
```

Comment:
Can we make this an option rather than the default?

This script is also used for correctness testing, so this will slow that down 30x.

Reply:
I removed the times=30. Functionally, line 239 does a loop with enough iterations already.